core/services/keystore/vrf_test.go

```diff
@@ -120,7 +120,7 @@ func Test_VRFKeyStore_E2E(t *testing.T) {
 		defer reset()
 
 		exportedKey := `{"PublicKey":"0xd2377bc6be8a2c5ce163e1867ee42ef111e320686f940a98e52e9c019ca0606800","vrf_key":{"address":"b94276ad4e5452732ec0cccf30ef7919b67844b6","crypto":{"cipher":"aes-128-ctr","ciphertext":"ff66d61d02dba54a61bab1ceb8414643f9e76b7351785d2959e2c8b50ee69a92","cipherparams":{"iv":"75705da271b11e330a27b8d593a3930c"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"efe5b372e4fe79d0af576a79d65a1ee35d0792d9c92b70107b5ada1817ea7c7b"},"mac":"e4d0bb08ffd004ab03aeaa42367acbd9bb814c6cfd981f5157503f54c30816e7"},"version":3}}`
-		importedKey, err := ks.Import([]byte(exportedKey), cltest.Password)
+		importedKey, err := ks.Import([]byte(exportedKey), "p4SsW0rD1!@#_")
```

Comment:
Should not these also use `testutils.Password`?

Reply:
No because the key is hardcoded in that string with the old password